src/button/styled-components.js

```diff
@@ -54,43 +57,92 @@ export const StartEnhancer = styled('div', ({$theme}: StylePropsT) => ({
   marginRight: $theme.sizing.scale500,
 }));
 
+export const LoadingSpinner = styled('div', ({$theme, $kind}: StylePropsT) => ({
+  height: $theme.sizing.scale600,
+  width: $theme.sizing.scale600,
+  borderRadius: '50%',
+  borderStyle: 'solid',
+  borderWidth: $theme.sizing.scale0,
+  borderTopColor:
+    $kind === KIND.primary ? $theme.colors.white : $theme.colors.primary,
+  borderLeftColor:
+    $kind === KIND.primary
+      ? 'rgba(255, 255, 255, 0.32)'
+      : 'rgba(39, 110, 241, 0.32)',
+  borderBottomColor:
+    $kind === KIND.primary
+      ? 'rgba(255, 255, 255, 0.32)'
+      : 'rgba(39, 110, 241, 0.32)',
+  borderRightColor:
+    $kind === KIND.primary
+      ? 'rgba(255, 255, 255, 0.32)'
+      : 'rgba(39, 110, 241, 0.32)',
```

Comment:
Maybe disabled button should have grey loader instead of white? White is a tad hard to see.

![image](https://user-images.githubusercontent.com/875591/45607571-3794ce00-ba02-11e8-8db3-3e7a2315dacb.png)

Reply:
@nguyensomniac do you have any specific recommendations for this case in terms of colors?

Reply:
+1 for grey—let's start with the same color as text